scripts/test_nearlib.sh

```diff
@@ -1,12 +1,16 @@
 #!/usr/bin/env bash
 set -ex
 
+SRC_DIR=$(pwd)/$(dirname "${BASH_SOURCE[0]}")
+export REPO_DIR=$(readlink -e "$SRC_DIR/..")
+pushd "${REPO_DIR}"
+
 export RUST_BACKTRACE=full
-./scripts/start_unittest.py --local &
+"${REPO_DIR}"/scripts/start_unittest.py --local &
```

Comment:
That’s not necessary (here and in other places). With `cd` you’ve changed directory to repo dir so `./scripts/whatever` is fine.

Reply:
In different places of this script, we are either in `${repo_dir}` then in some other directory. Using absolute path, make it less error prone.